test/api/students_api_test.rb

```diff
@@ -0,0 +1,66 @@
+require 'test_helper'
+require 'user'
+
+class StudentsApiTest < ActiveSupport::TestCase
+  include Rack::Test::Methods
+  include TestHelpers::AuthHelper
+  include TestHelpers::JsonHelper
+
+  def app
+    Rails.application
+  end
+
+  def test_get_students_with_authentication
+    # Create admin
+    adminUser = FactoryBot.create(:user, :admin) 
+
+    # Create unit
+    newUnit = FactoryBot.create(:unit)
+
+    #Create campus
+    newCampus = FactoryBot.create(:campus)
+
+    # Create student
+    studentUser = FactoryBot.create(:user, :student)
+
+    # Assign student to the unit
+    newUnit.enrol_student(studentUser, newCampus)
+
+    # The get that we will be testing.
+    get with_auth_token "/api/students/?unit_id=#{newUnit.id}",adminUser
+
+    response_keys = %w(first_name last_name student_id project_id)
+    last_response_body.each do | data |
+      userRes = User.find(data['student_id'])
```

Comment:
Hello Professor,
I want to obtain a user object from student_id or email because the JSON response does not have the id of the user.
Any suggestion for this.
I searched methods of user and unit but I did not find how to perform this.
@macite

Reply:
I think you should try to find the student users in `Project` class

Reply:
![Screen Shot 2020-01-26 at 8 45 28 am](https://user-images.githubusercontent.com/58777122/73127685-52c11600-4018-11ea-9c81-4252b957078c.png)
This should work @thisaraw

Reply:
It works. Thank you.

Reply:
I have not seen any update in this PR @thisaraw